pkg/i18n/english.go

```diff
@@ -906,7 +910,9 @@ func EnglishTranslationSet() TranslationSet {
 		LcRemoveRemotePrompt:                "Are you sure you want to remove remote",
 		DeleteRemoteBranch:                  "Delete Remote Branch",
 		DeleteRemoteBranchMessage:           "Are you sure you want to delete remote branch",
-		LcSetUpstream:                       "set as upstream of checked-out branch",
+		LcSetAsUpstream:                     "set as upstream of checked-out branch",
```

Comment:
we'll need to update the corresponding entries in other languages (not saying we need to translate everything here, just that we need to replace the `LcSetUpstream` key with `LcSetAsUpstream` in the other files

Reply:
I think that's done. Polish didn't have `LcSetUpstream` in the first place so I didn't touch that.